build/README.md

```diff
@@ -664,6 +668,14 @@ If you get this error when building Agones in WSL (`make build`, `make test` or
 - Are your project files on a different folder than C? If yes, then you should either move them on drive C or set up Docker for Windows to share your project drive as well
 - Did you set up the volume mount for Docker correctly? By default, drive C is mapped by WSL as /mnt/c, but Docker expects it as /c. You can test by executing `ls /c` in your linux shell. If you get an error, then follow the instructions for [setting up volume mount for Docker](https://nickjanetakis.com/blog/setting-up-docker-for-windows-and-wsl-to-work-flawlessly#ensure-volume-mounts-work)
 
+#### Error: cluster-admin-binding already exists
+
```

Comment:
As a separate PR, should we make the name of `cluster-admin-binding` unique to the user? (short Hash of the account name or something?)

Reply:
This I am not sure about and would have to look into that a bit more. Maybe someone with more Kubernetes knowledge can comment.

Reply:
Fair enough. I'll write up a small ticket for it - was just thinking out loud.

Otherwise the rest of this LGTM!